.github/pull_request_template.md

```diff
@@ -7,6 +7,7 @@ This PR ...
 ## Checklist
 - [ ] Tested locally?
 - [ ] Ran `quickTest all`?
+- [ ] Ran `./prime test` against local Docker ReportStream container?
```

Comment:
I think `./prime test` is now easy to use, and can become part of our canon of tests.